### DISCUSSION TITLE
What event is called when server is stopped?

### DISCUSSION THREAD

**Discussion by ShiftSad** [Minestom/Minestom#1709] [Help] [answered]
Hey, I need an event for when my server is closed, but it appears to not exist?

No stopped event 😟
https://javadoc.minestom.net/net/minestom/server/event/server/package-summary.html

**Comment by ZakShearman** [comment, marked as answer]
You should use a shutdown task: https://javadoc.minestom.net/net/minestom/server/timer/SchedulerManager.html#buildShutdownTask(java.lang.Runnable)

**Comment by emortaldev** [comment]
there is a few discussion answers on this lol

**Reply by ShiftSad** [reply]
Sorry, searched event, stopped, and some other stuff and couldn't find.
